tests/Shared/DistributedApplicationTestingBuilderExtensions.cs

```diff
@@ -21,6 +21,7 @@ public static IDistributedApplicationTestingBuilder WithTestAndResourceLogging(t
     {
         builder.Services.AddXunitLogging(testOutputHelper);
         builder.Services.AddLogging(builder => builder.AddFilter("Aspire.Hosting", LogLevel.Trace));
+        builder.Services.AddLogging(builder => builder.AddFilter("Aspire.Hosting.Dcp", LogLevel.Trace));
```

Comment:
Lets remmove this too. If you need it in your specific test cases you should be able to do it.